include/lsst/jointcal/AstroUtils.h

```diff
@@ -1,37 +1,15 @@
 // This may look like C code, but it is really -*- C++ -*-
-#ifndef USNOUTILS__H
-#define USNOUTILS__H
```

Comment:
This must be kept, but should be standardized to the [LSST convention](https://developer.lsst.io/coding/cpp_style_guide.html#header-files-must-include-a-construction-that-prevents-multiple-inclusion). Same with the `#endif` down below, obviously.